.eslintrc.json

```diff
@@ -25,6 +32,7 @@
 				"null": "never"
 			}
 		],
+		"no-constant-condition": ["error", { "checkLoops": false }],
```

Comment:
There's one `while (true)` loop left in `findDelimiterPairContainingSelection`. Figured those are probably more useful than the lint rule, and IMO `for (;;)` is ugly.